### PR TITLE
feat(@schematics/angular): add several prompts to ng-new

### PR DESCRIPTION
**Pull request by clydin**
Initial prompt usage when executing `ng new` for 7.0.

When executing `ng new` (no options are required now), the user will be prompted for the following:
* name of the project
* selector prefix for components
* whether they will use Angular routing
* stylesheet format (css, scss, etc.)

If the user provides the option on the command line (e.g., `--style scss`), the prompt for that option will not be displayed.

### CODE REVIEW

File: packages/schematics/angular/ng-new/schema.json [hunk 1]
```diff
@@ -95,19 +96,33 @@
     "routing": {
       "type": "boolean",
       "description": "Generates a routing module.",
-      "default": false
+      "default": false,
+      "x-prompt": "Will the project use Angular routing?"
```
Comment (StephenFluin) [inline packages/schematics/angular/ng-new/schema.json]: What tense & tone should be used for prompts?

It feels like this question should be more about what WE will do, like 'Would you like to generate a routing module?' or 'Should we add routing to the project?'

File: packages/schematics/angular/ng-new/schema.json [hunk 2]
```diff
@@ -95,19 +96,33 @@
...
     "style": {
       "description": "The file extension to be used for style files.",
       "type": "string",
-      "default": "css"
+      "default": "css",
+      "x-prompt": {
+        "message": "Select preferred style format for the project",
+        "type": "list",
+        "items": [
+          { "value": "css", "label": "CSS" },
```
Comment (StephenFluin) [inline packages/schematics/angular/ng-new/schema.json]: I think the Style options should be ordered by popularity, so:

1. css
2. scss
3. sass
4. less
5. stylus

File: packages/schematics/angular/ng-new/schema.json [hunk 3]
```diff
@@ -95,19 +96,33 @@
...
     },
     "prefix": {
       "type": "string",
       "format": "html-selector",
       "description": "The prefix to apply to generated selectors.",
+      "minLength": 1,
       "default": "app",
-      "alias": "p"
+      "alias": "p",
+      "x-prompt": "Choose an HTML selector prefix for the project:"
```
Comment (StephenFluin) [inline packages/schematics/angular/ng-new/schema.json]: I don't think we should prompt for a selector. I think `app-` is fine for most folks. Super open to other ideas on this one.

File: packages/schematics/angular/ng-new/schema.json (hunk 1 above)
Comment (clydin) [reply packages/schematics/angular/ng-new/schema.json]: Updated.

File: packages/schematics/angular/ng-new/schema.json (hunk 2 above)
Comment (clydin) [reply packages/schematics/angular/ng-new/schema.json]: Done.

File: packages/schematics/angular/ng-new/schema.json (hunk 3 above)
Comment (clydin) [reply packages/schematics/angular/ng-new/schema.json]: Removed.  Keeping it simple for now makes sense.  We can also add more later.

File: packages/schematics/angular/ng-new/schema.json [hunk 4]
```diff
@@ -95,19 +96,32 @@
     "routing": {
       "type": "boolean",
       "description": "Generates a routing module.",
-      "default": false
+      "default": false,
+      "x-prompt": "Would you like to generate a routing module?"
```
Comment (alexeagle) [inline packages/schematics/angular/ng-new/schema.json]: what happens if the user doesn't know what a "routing module" is?

**Pull request merged**